extensions/msal-node-extensions/package.json

```diff
@@ -34,7 +34,7 @@
     "lint": "cd ../../ && npm run lint:node:extensions",
     "lint:fix": "npm run lint -- -- --fix",
     "prepare": "npm install ../../lib/msal-common",
-    "prepack": "cd ../../lib/msal-common && npm install --production && cd ../../extensions/msal-node-extensions && npm run build:all"
+    "prepack": "npm run build:all && cd ../../lib/msal-common && npm install --production && cd ../../extensions/msal-node-extensions"
```

Comment:
Don't think I understand what's happening here...

Reply:
We are removing `msal-common` non-production dependencies before packing.

Reply:
Why do we need to install common's dependencies AFTER building it? And why do we then change back to the extensions directory? All the steps after the build feel unnecessary, what am I missing?